stories/multi-select.stories.js

```diff
@@ -26,6 +26,13 @@ const generateItems = size =>
     label: `Item ${index}`
   }));
 
+const generateItemsWithGroups = (size, groupSize) =>
```

Comment:
Consider splitting all these story utils to their own file to clean up the stories file.